client/packages/electron/README.md

```diff
@@ -42,10 +42,19 @@ In `packages/common`
 
 Since front end for client is server through remote server, it needs to be rebuilt if you are adding functionality that interacts with client, you can change  `ipcMain.on(IPC_MESSAGES.CONNECT_TO_SERVER` in `electron.ts`, see comment in code
 
-
 ### Debugigng
 
 Similar to [android](../android/README.MD#debugging), to speed up development we can serve both discovery and main web app through webpack server. Running `yarn start-local` and `yarn electron:start-local` from client would allow for hot reload and debugging typescript (can open developer tools with the same shortcut as chrome).
 
 Please note that when debugging with live reload, connections to a discovered server will always go through to the webpack server (regardless of which server is being selected). 
-Also the discovery may not show any servers when hot reloaded and may require stopping and starting electron
\ No newline at end of file
+Also the discovery may not show any servers when hot reloaded and may require stopping and starting electron
+
+### Self signed cert SSL security
+
+To avoid error being thrown by electron when certificate is self signed, we override certificate error listener and allow connection when:
```

Comment:
```suggestion
To avoid an error being thrown by electron when the certificate is self-signed, we override the  certificate error listener and allow connection when:
```